lib/interceptor.js

```diff
@@ -18,6 +18,11 @@ try {
 module.exports = Interceptor
 
 function Interceptor(scope, uri, method, requestBody, interceptorOptions) {
+  // Adds a leading slash if required
+  if (typeof uri === 'string' && !['', '*', '/'].includes(uri.charAt(0))) {
```

Comment:
Do we need the check if `uri` is a string?

For the condition, how about `/^[^/*]`/.test(uri)`?

Reply:
`/^[^/*]/.test(uri)` works like a charm. Thanks @gr2m.

@paulmelnikow, yeah, I'm adding a comment for that.

Reply:
Comment added.